Make.inc

```diff
@@ -891,6 +891,10 @@ OPENBLAS_DYNAMIC_ARCH:=0
 OPENBLAS_TARGET_ARCH:=ARMV7
 endif
 
+# on x86 make sure not to use 80 bit math when we want 64 bit math.
+ifeq ($(BUILD_ARCH),i386)
```

Comment:
Should be ARCH, but also, we support many settings for this, so should check `ISX86` and `BINARY` which are set above.